src/embind/embind.js

```diff
@@ -55,12 +55,12 @@ var LibraryEmbind = {
     // names. This lets the test suite know that.
     Module['DYNAMIC_EXECUTION'] = true;
 #endif
-#if ASSERTIONS
-    Module['ASSERTIONS'] = true;
-#endif
 #if EMBIND_STD_STRING_IS_UTF8
     Module['EMBIND_STD_STRING_IS_UTF8'] = true;
 #endif
+#if ASSERTIONS
+    Module['ASSERTIONS'] = true;
+#endif
```

Comment:
This code movement seems to have no effect - do we need it? (Is there some other benefit like aesthetics? Sorry if I'm missing that.)

Reply:
It's a stray change while I was trying to figure out some test problems. I can move it back, or would it be better to make it first, so the properties are alphabetically sorted?

Reply:
I don't have strong feelings - either way sgtm. Mainly I wanted to check if I was missing something.

Reply:
I made it alphabetical order, though I hope we never have enough such properties that it actually matters. Thanks for reviewing!